src/sourcerer/presentation/screens/main/main.py

```
@@ -182,9 +180,11 @@ async def init_storages_list(self) -> None:
                     credentials.uuid: storages,
                     **self.storage_list_sidebar.storages,
                 }
-            except Exception:
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)